cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/ClouderaManagerCommandPollerObject.java

```diff
@@ -7,15 +7,12 @@
 
 public class ClouderaManagerCommandPollerObject extends ClouderaManagerPollerObject {
 
-    private final Stack stack;
-
     private final ApiClient apiClient;
 
     private final BigDecimal id;
 
     public ClouderaManagerCommandPollerObject(Stack stack, ApiClient apiClient, BigDecimal id) {
         super(stack, apiClient);
-        this.stack = stack;
         this.apiClient = apiClient;
```

Comment:
You could have add some null checks here.

Reply:
I am not sure if it is a bad behaviour if the stack is null, so I checked the stack for null in the event service

Reply:
https://github.com/hortonworks/cloudbreak/pull/11449/files#diff-03a2afa3da231e8ffae47ea036184e2b9590359d2bb6b767abc3309864e38e88R126